_posts/2020-10-23-kubernetes-webhooks-made-easy-with-openfaas.md

```diff
@@ -0,0 +1,265 @@
+---
+title: "Kubernetes Webhooks made easy with OpenFaaS"
+description: "Extending the Kubernetes with Admission webhook using OpenFaaS Serverless Functions"
+date: 2020-10-23
+image: /images/2020-10-github-actions/lego.jpg
+categories:
+  - arkade
+  - kubectl
+  - faas-cli
+  - kind
+  - k8s
+  - admissionwebhooks
+  - validatingadmissionwebhooks
+  - k8s extensibility
+author_staff_member: developer-guy
+dark_background: true
+---
+
+Extending the Kubernetes with Admission webhook using OpenFaaS Serverless Functions
+
+<p align="center">
+<img height="128" src="/images/openfaas_light.png">
+</p>
+
+<p align="center">
+<img height="128" src="/images/2020-10-23-k8s-validatingwebhook-openfaas/heart.png">
+</p>
+
+<p align="center">
+<img height="128" src="/images/openfaas/kubernetes.png">
+</p>
+
+## Introduction to Kubernetes Admission webhooks
```

Comment:
In this section please give a short 1-liner on why you used openfaas for this over doing it with the microservice on its own.

Reply:
done.